truckersmp-cli

```diff
@@ -529,6 +531,25 @@ Please report an issue: {}""".format(e, URL.issueurl))
             sys.exit("Failed to download mod files.")
 
 
+def get_current_steam_user():
+    """
+    Get the current AccountName with saved login credentials.
+
+    If successful this returns the AccountName of the user with saved credentials.
+    Otherwise this returns None.
+    """
+    for path in File.loginusers_paths:
+        try:
+            login_vdf = vdf.parse(open(path))
+
+            for _, info in login_vdf["users"].items():
+                if info["MostRecent"] == "1" and info["RememberPassword"] == "1":
```

Comment:
My `loginusers.vdf` has an item named `mostrecent` instead of `MostRecent`.

```
>>> login_vdf
{'users': {'xxx': {'AccountName': 'myname', 'PersonaName': 'myname', 'RememberPassword': '1', 'mostrecent': '1', 'Timestamp': 'yyy', 'WantsOfflineMode': '0'}}}
```

Does your `loginusers.vdf` have `MostRecent` item? *If does not*, we have to replace the key `MostRecent` with `mostrecent`.

Reply:
> Does your `loginusers.vdf` have `MostRecent` item?

Yes, it's upper case for me:
~~~
		"RememberPassword"		"1"
		"MostRecent"		"1"
~~~

> _If does not_, we have to replace the key `MostRecent` with `mostrecent`.

Why? Then it fails for me. I added tests for both possible values.

Reply:
> Yes, it's upper case for me:

This was *completely unexpected*...

> Why? Then it fails for me.

Sorry, I *assumed* that it's always lowercase *for some reasons*.

> I added tests for both possible values.

Thanks, but other items contain uppercase chars (e.g. `AccountName`) in my `loginusers.vdf`:

Similar cases (Only `mostrecent` is "all lowercase"):

* https://github.com/shravan2x/Gameloop.Vdf/issues/14
* https://github.com/TcNobo/TcNo-Acc-Switcher/wiki
* https://gist.github.com/l3laze/4c6f5061d74aac15347b2daba56c6061

I'm not sure whether we *really need to do that for other keys*.

Reply:
At least my file is consistent with its capitalization. :)
Removed the other checks.

Reply:
> Removed the other checks.

Thanks.